doc/source/tune/_tutorials/tune-usage.rst

```diff
@@ -396,6 +396,8 @@ If a string is provided, then it must include replacement fields ``{source}`` an
         sync_process = subprocess.Popen(sync_cmd, shell=True)
         sync_process.wait()
 
+By default, syncing occurs every 300 seconds. To change the frequency of syncing, set the ``TUNE_CLOUD_SYNC_S`` environment variable in the driver to the desired syncing period. Note that the ``global_checkpoint_period`` argument also affects the frequency of uploading.
```

Comment:
Can you say more about how global_checkpoint_period affects frequency of uploading?

Reply:
added some more clarification here